### PR TITLE
Assignment45 prem

### PR DESCRIPTION
**Pull request by Premlata1726**
image

### CODE REVIEW

File: src/prem/Assignment45.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package prem;
+
+public class Assignment45 {
+	
+	void pattern() {
+		for (int i =1; i<=4;i++){
+			if(i%2==0) {
+				System.out.println("#*#");
```
Comment (AnvitP) [inline src/prem/Assignment45.java:8]: You cannot directly print the string... kindly write appropriate logic

Comment (Premlata1726) [reply src/prem/Assignment45.java]: image

Corrected the logic, please review

Comment (AnvitP) [reply src/prem/Assignment45.java]: Still You are directly printing the string... You need to write appropriate logic for printing * and hash

Comment (Premlata1726) [reply src/prem/Assignment45.java]: image

Corrected logic without directly pointing to string

**Pull request merged**